wallet/res/values/strings.xml

```diff
@@ -372,7 +372,7 @@
     <string name="activity_security_backup_wallet">Backup Wallet to File</string>
     <string name="activity_security_view_recovery_phrase">View Recovery Phrase</string>
     <string name="activity_security_advanced">Advanced Security</string>
-    <string name="activity_security_biometric_auth">Biometric Authentication</string>
+    <string name="activity_security_biometric_auth">Fingerprint Authentication</string>
```

Comment:
It can be also face auth, isn't it? So I guess biometric fits better here
The rest looks good

Reply:
This app currently only supports Fingerprint authentication.